asm/interpreter.asm

```diff
@@ -78,9 +78,18 @@ quit_reset
     stx     SOURCE_ID_MSB
     stx     SAVE_INPUT_STACK_DEPTH
 
-    jsr     close_all_logical_files
-    jsr     CLRCHN
-
+; Used registers: A, X, Y
+close_all_logical_files:
+    jsr CLRCHN
+    ldx #0
+-   txa
+    pha
+    jsr CLOSE
```

Comment:
is it safe to call CLRCHN before CLOSE? Usually, CLOSE is called before CLRCHN?

Reply:
Maybe the order does not matter? I do not remember anymore how this works :-)

Reply:
You should call CLRCHN before CLOSE. Especially if the device you want to CLOSE is a current device.
 Otherwise, one of your current devices will be invalid after close.

Reply:
It is also true that these fix no problems, and the memory savings are likely inconsequential.